src/renderers/index.ts

```diff
@@ -31,7 +32,8 @@ export
 class HTMLWidget extends Widget {
   constructor(html: string) {
     super();
-    this.node.innerHTML = html;
+    let range = document.createRange();
+    this.node.appendChild(range.createContextualFragment(html));
```

Comment:
Go ahead and merge this in for now. When we get to using rendermime in dashboard server, we can send another PR.